src/apps/vpn/models/subscriptiondata.cpp

```diff
@@ -222,6 +224,16 @@ void SubscriptionData::writeSettings() {
   SettingsHolder::instance()->setSubscriptionData(m_rawJson);
 }
 
+bool SubscriptionData::plusTax() {
+  MozillaVPN* vpn = MozillaVPN::instance();
+  QString countryCode = vpn->location()->countryCode();
+
+  connect(vpn->location(), &Location::changed,
```

Comment:
This is exactly the bug I'm talking about. The +tax is based on the location at the subscription time.
Guardian should have this data.

Reply:
@brizental , in case Guardian does not have it can we use a global state prop set at the subscription time?

Reply:
Yes we can. This feels like something Guardian should have though, if it doesn't maybe we should change it in Guardian to provide the "+ tax".